src/components/ImageView/index.native.js

```diff
@@ -102,21 +144,14 @@ class ImageView extends PureComponent {
                         this.imageZoomScale = scale;
                     }}
                 >
-                    <ImageWithSizeCalculation
```

Comment:
hmmm I wonder if we should move all this logic to this component instead... what's the value of keeping both?

Reply:
`ImageWithSizeCalculation` component is used in Web & Desktop platforms as well and current implementation works just fine for those. Keeping the logic contained in ImageView native implementation, we basically keep the unnecessary dependencies out of web bundle & avoid webpack errors. If you still prefer to keep it in ImageSizeWithCalculation then it needs to have a native implementation which uses FastImage & RNImageSize.

Reply:
Oh ok ok, no, this is good.